PRIVACY.md

```diff
@@ -181,6 +182,7 @@ r2c will:
 |             | Warnings                                | Array of Warning Classes (compile-time-constant)                       | Understand most common warnings users encounter                                            | `["TimeoutExceeded"]`                                                                                                                                                                 | WarningClass[] |
 |             |                                         |                                                                        |                                                                                            |                                                                                                                                                                                       |                |
 | Value       |                                         |                                                                        |                                                                                            |                                                                                                                                                                                       |                |
+|             | Engine requested                        | The engine requested by the user                                       | Understand what engines people are using and debug problems                                | `Interfile`                                                                                                                                                                           | str            |
```

Comment:
i'm not in love with this description (on the far right column), i'd prefer some mention of "debug engine-specific problems", perhaps